scripts/sagas/session.js

```diff
@@ -47,12 +47,34 @@ export function* listBuckets(getState, action) {
     const responses = yield call([client, client.batch], (batch) => {
       for (const {id} of data) {
         batch.bucket(id).listCollections();
+        // XXX future: query the /permissions endpoint to retrieve the other
+        // collections we may have access to bcause they're shared with us
       }
     });
     const buckets = data.map((bucket, index) => {
       const collections = responses[index].body.data;
       return {id: bucket.id, collections};
     });
+    const permissions = yield call(requestPermissions);
```

Comment:
Having a `if` here for a (capability/protocol version) check wouldn't be too much hassle actually